script/check-collections.js

```diff
@@ -0,0 +1,53 @@
+/* eslint-disable no-console */
+
+const defaultCollections = require('../script/collections/default.json');
+const brandConsolidationCollections = require('../script/collections/brand-consolidation.json');
+
+const checkCollections = buildOptions => (files, metalsmith, done) => {
+  const collection = buildOptions['brand-consolidation-enabled']
+    ? Object.keys(brandConsolidationCollections)
+    : Object.keys(defaultCollections);
+
+  const brokenCollections = Object.keys(files).reduce((accum, key) => {
```

Comment:
What made you decide to use `.reduce`? This seems like an instance where it may be simpler to `.map` the files array into an array of errors messages, then `.filter` out any `null` values.

Reply:
@ncksllvn That is what I am doing with the reduce. I am creating a map and filter in the reduce. I thought it would be simpler to do it in 1 operation instead of 2.